apps/src/applab/designMode.js

```diff
@@ -280,24 +309,36 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging) {
 };
 
 designMode.onClear = function() {
+  // TODO (brent) - have this clear just the current screen instead of everything
```

Comment:
I'm not sure we need the 'clear' functionality you're describing. may be worth checking with Sarah before spending time on this TODO.